docs/11.1/api/hooks.md

```diff
@@ -1662,21 +1648,26 @@ Fired by [TrimRows](@/api/trimRows.md) plugin after untrimming rows. This hook i
 
 ### afterUpdateData
   
-::: source-code-link https://github.com/handsontable/handsontable/blob/0472af66268f29ceb64d1f046b74a05149cffe8d/../handsontable/src/pluginHooks.js#L309
+::: source-code-link https://github.com/handsontable/handsontable/blob/5bef6a42994fcc395de546c0f8adbc297579650d/../handsontable/src/pluginHooks.js#L316
```

Comment:
How is this tested? These links in the docs do not work, as reported https://github.com/handsontable/handsontable/issues/8948. We can fix this when a new person joins to help us work on the automations.

Reply:
@warpech - should I manually update the links for now?

Reply:
I don't know, I think it doesn't matter until #8948 is fixed.